lib/Ohmbrewer_Rhizome.h

```diff
@@ -247,6 +253,17 @@ namespace Ohmbrewer {
          * @return Error or success code, according to the requirements specified by addSprout
          */
         int addRIMS(char* params);
+
+        /**
+         * Adds a sprout to the list and rebuids the index
+         * @param equipment The sprout being added
+         */
+        void saveNewSprout(Equipment* sprout);
```

Comment:
I'm not sure I understand the logic behind naming it the same though. Why is saveNewSprout not adequate?